utils/tracking.js

```diff
@@ -6,13 +6,13 @@ if(process.env.NEXT_PUBLIC_GA_ID) {
 }
 
 export const trackPageView = (url) => {
-    if(getCookiesFromLocalStorage && process.env.NEXT_PUBLIC_GA_ID) {
+    if(getCookiesFromLocalStorage() && process.env.NEXT_PUBLIC_GA_ID) {
```

Comment:
@sarahgreen I realized that I didn't actually make the method call here, so it would have been tracking whether or not the user agreed to cookies. This should fix it 🤦🏻‍♀️ I always think I'm writing Ruby.